File: 5/2. Линейный поиск/D. Шахматная доска/mike.py
from tests.index import test, input
test(1)

__input__ = input
def input():
  try: return __input__()
  except: return None

n = int(input())

board = [ [ True ] * 10 for _ in range(10) ]
for _ in range(n):
  x, y = map(int, input().split())
  board[x][y] = False
  

p = 0
for x in range(1, 10):
  for y in range(1, 10):
    p += int(board[x][y - 1] != board[x][y])
    p += int(board[x - 1][y] != board[x][y])
  
print(p)

File: 6/3. Стеки, очереди, деки/A. Правильная скобочная последовательность/mike.js
const fs = require('fs')
const test = 1
const __input__ = fs.readFileSync(`${__dirname}/tests/${test}`, 'utf8').split('\n')
const __ans__ = fs.readFileSync(`${__dirname}/tests/answers/${test}`, 'utf8').trim()

const input = ((i = 0) => () => __input__[i++])()

// COPY FROM HERE and uncomment this
/*
const fs = require('fs')
const __input__ = fs.readFileSync('input.txt', 'utf8').split('\n')
const input = ((i = 0) => () => __input__[i++])()

//*/
const s = input()
const check = (s) => {
  const closing = { ')': '(', '}': '{', ']': '[' }
  const stack = []
  for (let c of s) {
    if (c in closing) {
      if (stack.pop() !== closing[c]) return false
    } else {
      stack.push(c)
    }
  }

  return !stack.length
}

console.log(check(s) ? 'yes' : 'no')
console.log(__ans__)
/*
fs.writeFileSync('output.txt', `${check(s) ? 'yes' : 'no'}`)
*/
